src/requestCodeGen.ts

```diff
@@ -114,10 +114,19 @@ export function requestCodeGen(paths: IPaths, options: ISwaggerOptions): string
         pathReplace = parsedParameters.requestPathReplace
       }
 
-      let responseType =
-        v.responses['200'] && v.responses['200'].schema && v.responses['200'].schema.$ref
-          ? refClassName(v.responses['200'].schema.$ref)
-          : 'any'
+      let responseType: string;
+
+      if (!v.responses['200'] || !v.responses['200'].schema) {
+        responseType = 'any';
+      } else if (v.responses['200'].schema.$ref) {
+        responseType = refClassName(v.responses['200'].schema.$ref)
+      } else {
+        responseType = v.responses[200].schema.type;
+        if (responseType == 'object' || responseType == 'array') {

```

Comment:
if return a Array ,I think  schema like this
```js
schema:{
  items:{
     type:'string'
  }
}
```
then, I think you give it a try

Reply:
Hi, thank you for the review.
To my understanding, It can take the `type: "array"` parameter explicitly
and implicit array type is also allowed (needs `items` property) in Swagger spec.

I changed the code to handle the schema that is defined as array implicitly.